### DISCUSSION TITLE
if u have goguardian, my new exploit might help

### DISCUSSION THREAD

**Discussion by schoolexploitkid** [3kh0/ext-remover#1233] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
it abuses text servers so that the blocker will not know when to block. havent been tested yet, so feel free to leave comments.
https://github.com/schoolexploitkid/UNBLOCK3R

**Comment by WalkTheEarth** [comment]
NOT WORKING
Tested on Lightspeed (Relay version 3.7.3.1728417270, ChromeOS v130)

**Reply by schoolexploitkid** [reply]
it might not work for non-goguardian filter, also this has been patched

**Comment by akabutnicer** [comment]
ah yeah i found this a year ago, glad someone actually made it into an exploit

**Comment by ghost** [comment]
nope, does not work just shows me an 404 error message

**Reply by schoolexploitkid** [reply]
rip they might have patched it

**Reply by ghost** [reply]
ik bro

**Reply by ghost** [reply]
do u have any other exploits goguardian blocking everything

**Reply by schoolexploitkid** [reply]
try reloading the page if it says error too long url
